model/DBHelper.cs

```diff
@@ -53,11 +55,21 @@ public class DBHelper {
 			}
 		}
 
-		public static void CreateDb(string conn) {
-			var cnBuilder = new SqlConnectionStringBuilder(conn);
+		public static void CreateDb(string connection, string databaseFilesPath = null) {
+			var cnBuilder = new SqlConnectionStringBuilder(connection);
 			var dbName = cnBuilder.InitialCatalog;
 			cnBuilder.InitialCatalog = "master";
-			ExecSql(cnBuilder.ToString(), "CREATE DATABASE [" + dbName + "]");
+		    var files = string.Empty;
+		    if (databaseFilesPath != null) {
+		        Directory.CreateDirectory(databaseFilesPath);
```

Comment:
This function is actually smart about that

http://stackoverflow.com/a/9065716